webui/src/App.js

```diff
@@ -204,10 +236,8 @@ function ValueMonitor(props) {
       ctx.fillRect(0, threshold_pos-Math.floor(threshold_height/2), canvas.width, threshold_height);
 
       // Threshold Label
-      const thresholdLabel = thresholdLabelRef.current;
-      thresholdLabel.innerHTML = kCurThresholds[index];
+      thresholdLabel.innerText = kCurThresholds[index];
       // This is default React CSS font style.
```

Comment:
Move this comment above to where bodyFontFamily is defined

Reply:
done